src/lang/de.rs

```diff
@@ -407,6 +407,6 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("Search", "Suchen"),
         ("Closed manually by the web console", "Manuell über die Webkonsole beendet"),
         ("Local keyboard type", "Lokaler Tastaturtyp"),
-        ("Select local keyboard type", "Lokalen Tastaturtyp Auswählen"),
+        ("Select local keyboard type", "Lokalen Tastaturtyp auswählen"),
```

Comment:
L... T... a... looks strange, though i do not know German.

Reply:
In German verbs are basically written in lower case:

Quote from [https://de.wiktionary.org/wiki/Verzeichnis:Deutsch/Rechtschreibung/Gro%C3%9F-_und_Kleinschreibung_von_Verben/2006#.C3.9Cbersicht]

Verben werden grundsätzlich kleingeschrieben.

    Beispiele: schreiben, tanzen, ich denke, wir haben gelesen ...